data/sql/db-world/as_2024_03_17_00.sql

```diff
@@ -21,4 +21,10 @@ INSERT INTO `npc_text` (`ID`, `text0_0`) VALUES (@entryNpcText1, @text1);
 SET @entryNpcText2 := 1000004;
 SET @text2 := '                                |TInterface/ICONS/achievement_bg_killXEnemies_generalsroom:25|t$B                        Solo 3v3 Arena$B$B              Melee Caster Healer [|cffff0000off|r]';
 DELETE FROM `npc_text` WHERE `ID` = @entryNpcText;
-INSERT INTO `npc_text` (`ID`, `text0_0`) VALUES (@entryNpcText2, @text2);
\ No newline at end of file
+INSERT INTO `npc_text` (`ID`, `text0_0`) VALUES (@entryNpcText2, @text2);
+
+-- Command
+DELETE FROM `command` WHERE `name` IN ('qsolo', 'testqsolo');
+INSERT INTO `command` (`name`, `security`, `help`) VALUES
+('qsolo', 0, ''),
+('testqsolo', 4, '');
```

Comment:
adding a newline at the end
```suggestion
('testqsolo', 4, '');

```